README.md

```diff
@@ -533,8 +533,9 @@ $ ./bin/test
 
 If you are on a Mac, you may also test the OSX Keychain provider:
```

Comment:
Not for this PR, but I wonder if there are online testing services to provide CI pipelines with mac envs, for situations like this, that we could use and thereby move this into the automated bucket?